WeatherApp/WebAPI/Jobs/UpdateWeatherHistoryJob.cs

```diff
@@ -0,0 +1,21 @@
+﻿using BL;
+using Quartz;
+
+namespace WebAPI.Jobs
+{
+    public class UpdateWeatherHistoryJob : IJob
+    {
+        private readonly IWeatherHistoryService _weatherHistoryService;
+
+        public UpdateWeatherHistoryJob(IWeatherHistoryService weatherHistoryRepository)
+        {
+            _weatherHistoryService = weatherHistoryRepository;
+        }
+
+        public async Task Execute(IJobExecutionContext context)
+        {
+            var cityNames = (IEnumerable<string>)context.MergedJobDataMap["cityNames"];
```

Comment:
Check for a null. move literal of "cityNames" to the some constants

Reply:
Added null check and moved "cityNames" to separate constants file.